src/modules/hero-content/HeroContent.pug

```diff
@@ -0,0 +1,8 @@
+picture
+    source(media="(max-width: 639px)" srcset=images.small.url)
```

Comment:
Instead of hardcoding these numbers, you can send an object to the template that can hold that data. The best way would be to create a new object that can have data related to the Hero itself, and merge it with data from the breakpoints. If you need more info about this, please let me know.